packages/core/src/actions/account/wallet/send-funds-to-remote.ts

```diff
@@ -0,0 +1,60 @@
+import { IbcClientTypes, ManagerClient } from '../../../codegen/abstract'
+import { MaybeArray } from '../../../types/utils'
+import { WithArgsAndCosmWasmSignOptions } from '../../../types/with-args'
+import { Asset, assetToNativeAsset } from '../../../utils/assets'
+import { executeIbcAction } from './execute-ibc-action'
+import { BaseWalletParameters } from './types'
+
+export type SendFundsToRemoteParameters = Omit<
+  WithArgsAndCosmWasmSignOptions<
+    BaseWalletParameters & {
+      hostChainName: string
+      assets: MaybeArray<Asset>
```

Comment:
Not sure if this should only be native assets since it only accepts native assets?